baseplate/clients/sqlalchemy.py

```diff
@@ -254,12 +282,30 @@ def on_after_execute(
         conn.info["span"].finish()
         conn.info["span"] = None
 
+        labels = {
+            "host": conn.engine.url.host,
+            "database": conn.engine.url.database,
+            "success": "true",
+        }
+
+        self.requests_total.labels(**labels).inc()
+        self.latency_seconds.labels(**labels).observe(perf_counter() - self.time_started)
+
     def on_error(self, context: ExceptionContext) -> None:
         """Handle the event which happens on exceptions during execution."""
         exc_info = (type(context.original_exception), context.original_exception, None)
         context.connection.info["span"].finish(exc_info=exc_info)
         context.connection.info["span"] = None
 
+        labels = {
+            "host": context.connection.engine.url.host,
+            "database": context.connection.engine.url.database,
+            "success": "false",
+        }
+
+        self.requests_total.labels(**labels).inc()
```

Comment:
I like passing a kwarg dict here, it keeps values explicitly tied to their keys so ordering doesn't matter as much.